src/test/mocks/VatMock.sol

```diff
@@ -79,6 +80,18 @@ contract VatMock {
         urns[i][u] = urn;
     }
 
+    function grab(bytes32 i, address u, address v, address w, int256 dink, int256 dart) external {
+        Urn storage urn = urns[i][u];
+
+        urn.ink = add(urn.ink, dink);
+        urn.art = add(urn.art, dart);
+
+        int256 dtab = mul(RAY, dart);
+
+        gem[i][v] = sub(gem[i][v], dink);
+        sin[w]    = sub(sin[w],    dtab);
+    }
```

Comment:
Don't think we need this function anymore as the `suck` + `grab` was removed. Not a huge deal if you want to leave it though.